python/orca/test/bigdl/orca/learn/spark/test_tf2_estimator.py

```diff
@@ -41,7 +41,7 @@ def compile_args(config):
     else:
         lr = 1e-3
     args = {
-        "optimizer": tf.keras.optimizers.SGD(lr),
+        "optimizer": tf.keras.optimizers.Adam(lr),
```

Comment:
SGD will possibly get gradient disappear and get nan weights; thus changed to adam